step-templates/RavenDB - Create File System

```diff
@@ -0,0 +1,103 @@
+  },
+  "SensitiveProperties": {},
+  "Parameters": [
+    {
+      "Name": "ravenFileSystemURL",
+      "Label": "URL of RavenDB",
+      "HelpText": "URL of the RavenDB, where you would like the file system to be created. \n\nFor example **http://localhost:8080/**",
+      "DefaultValue": "http://localhost:8080/",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Name": "ravenFileSystemName",
+      "Label": "Name of the File System",
+      "HelpText": "Name the new File System will be called. Make sure the name is not being used already within this instance of RavenDB",
+      "DefaultValue": null,
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Name": "ravenActiveBundles",
+      "Label": "Active Bundles",
+      "HelpText": "A settings option, Active Bundles refers bundles that RavenDB has (Replication;Versioning; etc)",
+      "DefaultValue": null,
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Name": "ravenStorageTypeName",
+      "Label": "Name of the Storage Type",
+      "HelpText": "What storage type to use",
+      "DefaultValue": "Esent",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Select",
+        "Octopus.SelectOptions": "Value1|Display Esent\nValue2|Display Voron"
```

Comment:
The values should be `Esent` and `Voron` instead of `Value1` and `Value2`, and same question regarding the display values as above.